### PR TITLE
feat(web): packages

### PR DESCRIPTION
**Pull request by m8vago**
- Add version chains
- Add packages

### CODE REVIEW

File: web/crux-ui/locales/en/common.json [hunk 1]
```diff
@@ -95,6 +95,7 @@
   "pipelines": "Pipelines",
   "storages": "Storages",
   "configBundles": "Config bundles",
```
Comment (polaroi8d) [inline web/crux-ui/locales/en/common.json:97]: If you changed "config bundles" to singular, why are you leaving this as plural?

Comment (m8vago) [reply web/crux-ui/locales/en/common.json:97]: Cause this is the translation for the side-menu, the other is the `routes.ts`.

Comment (polaroi8d) [reply web/crux-ui/locales/en/common.json:97]: Okey, thanks for the clarification.

**Review by polaroi8d — APPROVED** [review]
Not tested in locally, but the code looks okey for me.

**Comment by m8vago** [comment]
> Not tested in locally, but the code looks okey for me.

I don't like the current schema, gonna clarify the version relations a bit.

File: web/crux/src/app/version/version.service.ts [hunk 2]
```diff
@@ -445,6 +458,10 @@ export default class VersionService {
     }
 
     const newVersion = await this.prisma.$transaction(async prisma => {
+      if (!parentVersion.chain) {
+        // we need to create the chain
```
Comment (robot9706) [inline web/crux/src/app/version/version.service.ts:462]: todo for future?

**Review by robot9706 — COMMENTED** [review]
1 minor comment

File: web/crux/src/app/version/version.service.ts (hunk 2 above)
Comment (m8vago) [reply web/crux/src/app/version/version.service.ts:462]: nope. forget to implement it. thanks!

**Pull request merged**